src/resources/troubleshooting.md

```diff
@@ -0,0 +1,720 @@
+#### Ghost Command
+Not many people encounter this problem, but it's still worth mentioning.\
+This problem is related to database synchronization (between your application data and your bot's data in the database).\
+**Solution Options**\
+*The 1st Solution*\
+Restart the app.\
+Close the app from Recent apps and reopen it. This may result in a deleted command that the deletion didn't sync with the database, and then you can delete that command again.\
+*The 2nd Solution*\
+Attempting to forcibly restart the bot.\
+You can do this in the settings of your bot in the app. Restarting the bot can send a retransfer of data from the application to the database, and then the deleted command will be deleted for real.\
+*The Solution Options Didn't Help*\
+In this case, join the [support server](https://discord.gg/botdesigner), create a ticket using the `!new` command and tell the staff your bot's ID.\
+The staff will mention one of the developers to take a look at this problem with your bot.
+
+### The Bot Takes A Long Time To Respond
+**The 1st Reason**\
+Your code is too long and/or complicated.\
+This may be the obvious reason if it is. Executing large and complex code takes more resources.\
+When writing code, you should think about how to make it more compact and less complex, but in a way that makes your idea come to fruition.
+
+**The 2nd Reason**\
+Your bot was rate limited. This can happen because of excessive requests to Discord API, performed by BDFD functions (such as `$addEmoji` and others).\
+You should not stack such functions and try to perform them all at once.
+
+**The 3rd Reason**\
+The node your bot is running on is experiencing some slight problems (such as a rate limits). If you're sure this is the case, you can regenerate the bot's token and replace it with a new one. This will cause your bot to change its node.
+
+### The Slash Commands Doesn't Appear
+**The 1st Reason**\
+Old version of the application. Make sure that you have a latest version of the application installed.\
+New versions of the application have been improved and updated, and new features have been added. In addition, errors with the validity check of the slash commands have been fixed. You will be warned if you set up your slash command incorrectly, in which case the application will not allow you to save the slash command.
+
+**The 2nd Reason**\
+Slash commands are cached by discord, so it takes time before they appear in discord.
+
+They're also cached on the client side, if they were successfully cached in discord. If other users have a new slash command and you don't, restart the discord client.\
+Restarting will cause existing slash commands to be cached.
+
+**The 3rd Reason**\
+Conflict of slash commands due to other services that you're no longer using. For example, if two slash commands have the same name, but one is created using a third-party service, and the other through the BDFD, this can cause a conflict and the slash command will not appear.\
```

Comment:
```suggestion
Conflict of slash commands due to other services that you're no longer using. For example, if two slash commands of same type have the same name, but one is created using a third-party service, and the other through the BDFD, this can cause a conflict and the slash command will not appear.\
```